### ISSUE TITLE
Rosetta emulation of amd64 version of ibm-semeru-runtimes:open-11-jdk-focal failed to load: j9vm29

### ISSUE THREAD

**Issue by elmnic** [ibmruntimes/Semeru-Runtimes#70]
Since upgrading to macOS 14.3 on an M2 mac and running Docker Desktop 4.27.2 with Rosetta emulation enabled, running `ibm-semeru-runtimes:open-11-jdk-focal` as an amd64 image fails with the output: 

```bash
docker run --rm --platform=linux/amd64 ibm-semeru-runtimes:open-11-jdk-focal java --version
libjvm.so preloadLibrary(/opt/java/openjdk/lib/default/libj9vm29.so): /opt/java/openjdk/lib/default/libj9vm29.so: cannot enable executable stack as shared object requires: Invalid argument
libjvm.so failed to load: j9vm29
```
vs 
```bash
docker run --rm --platform=linux/arm64 ibm-semeru-runtimes:open-11-jdk-focal java --version
openjdk 11.0.21 2023-10-17
IBM Semeru Runtime Open Edition 11.0.21.0 (build 11.0.21+9)
Eclipse OpenJ9 VM 11.0.21.0 (build openj9-0.41.0, JRE 11 Linux aarch64-64-Bit Compressed References 20231122_825 (JIT enabled, AOT enabled)
OpenJ9   - 461bf3c70
OMR      - 5eee6ad9d
JCL      - 0fd7a05277 based on jdk-11.0.21+9)
```

**Comment by MPV** [comment]
See also related issue over at:
- https://github.com/docker/for-mac/issues/7180

Docker says they (and hopefully soon Apple too) are working on it:
>This is a regression in Rosetta, on Apple's side. macOS Sonoma 14.3 ships with this regression. We'll contact Apple to see when they can fix it. For now, we've prepared a workaround we're planning on shipping with Docker Desktop 4.28.0.

**Comment by elmnic** [comment]
Apple has released `macOS 14.4` officially now and it includes the fix for this regression.
